src/fragments/fragments.test.ts

```diff
@@ -85,9 +85,7 @@ describe("interpretFragments", () => {
 describe("errorMessageHandling", () => {
   it("should return all errors when fragments have multiple errors", () => {
     expect(errorMessageHandling(whenDocumentHashInvalidAndNotIssued as VerificationFragment[])).toStrictEqual([
-      "HASH",
-      "IDENTITY",
-      "ISSUED",
+      "INVALID",
```

Comment:
Just thinking if this test is to check that multiple errors will be returned, why do we change it from the 3 errors to a single `INVALID` error?